src/main/java/com/frostwire/jlibtorrent/SessionManager.java

```diff
@@ -1119,6 +1119,10 @@ public void run() {
         t.start();
     }
 
+    public TorrentStats trackStats(TorrentHandle torrentHandle, long samplingIntervalInMs, long maxHistoryInMs) {
```

Comment:
I think having this for all torrents potentially helps the UI. For example, our clients perform a status request every second for every torrent. If `SessionManager` keeps this stats already calculated (the onces that we will be using anyway) it is less costly overall. The history can be enabled at request, depending on if you want to have a meaningful graph from the start or not.

I want to mention that when the times come, special attention should be paid to the denseness nature of the circular buffers to keep in memory. But let's wait to that moment.